src/app/modules/login/services/azure.ad.b2c.service.ts

```diff
@@ -27,6 +34,8 @@ export class AzureAdB2CService {
   }
 
   logout() {
+    this.cookieService.delete('msal.idtoken');
+    this.cookieService.delete('msal.idtoken');
```

Comment:
should we do it twice?

Reply:
No, I already corrected it, the one to be deleted is the 2nd cookie